user-metrics/mdservice/config.yaml

```diff
@@ -0,0 +1 @@
+port: 8080 # Port where the discovery service is hosted
```

Comment:
what about account, host, username, etc..
the same as it was in plugin?

Reply:
the discovery service will only work as an spcs service and every spcs service has the oauth token associated with it. So I was wondering, why not keep it simple and use the token to execute queries.

I can add password based auth too

Reply:
okay, lets keep it as it is